### PR TITLE
Nodejs updates to merge

### PR DESCRIPTION
**Pull request by kwek20**
# Description of change

Stuff that we missed or was needed. Part 1

### CODE REVIEW

**Review by DaughterOfMars — DISMISSED** [review]

**Review by DaughterOfMars — DISMISSED** [review]

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
CI fails

File: bindings/nodejs/lib/utils/milestone-helper.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+/**
+ * Compute a milestoneId from a milestone payload.
+ * @param payload The milestone payload.
+ * @returns The milestone id hex prefixed string.
+ */
+export function milestoneIdFromMilestonePayload(
+    payload: MilestonePayload,
+): string {
+    return Utils.milestoneId(payload);
+}
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/utils/milestone-helper.ts]: This feels really useless, I'd just let them call our function

File: bindings/nodejs/package.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@types/node": "^18.15.12",
+        "blakejs": "^1.2.1",
```
Comment (thibault-martinez) [inline bindings/nodejs/package.json]: why

File: bindings/nodejs/lib/utils/milestone-helper.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+/**
+ * Compute a blockId from a milestone payload.
+ * @param protocolVersion The protocol version to use.
+ * @param payload The milestone payload.
+ * @returns The blockId of the block with the milestone payload.
+ */
+export function blockIdFromMilestonePayload(
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/utils/milestone-helper.ts]: Idk, maybe just me but I wouldn't provide this, they can do it themselves, it seems oddly specific

File: bindings/nodejs/package.json (hunk 2 above)
Comment (kwek20) [reply bindings/nodejs/package.json]: needed lzater, ill add it then

File: bindings/nodejs/lib/utils/milestone-helper.ts (hunk 1 above)
Comment (kwek20) [reply bindings/nodejs/lib/utils/milestone-helper.ts]: sure

File: bindings/nodejs/lib/utils/milestone-helper.ts (hunk 3 above)
Comment (kwek20) [reply bindings/nodejs/lib/utils/milestone-helper.ts]: eh, its utils, theyre always specific, but removed

**Review by thibault-martinez — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Pull request merged**